code/game/gamemodes/endgame/supermatter_cascade/universe.dm

```diff
@@ -11,9 +11,13 @@
 	return 0
 
 /datum/universal_state/supermatter_cascade/OnTurfChange(var/turf/T)
-	var/turf/space/spess = T
-	if(istype(spess))
-		spess.overlays += "end01"
+	if(T.name == "space")
+		T.overlays += "end01"
+		T.underlays -= "end01"
+	else
+		T.overlays -= "end01"
```

Comment:
well the problem is that basically all our shuttles and ferries corner turfs in the game actually have space drawn on their sprite rather than having it as an underlay, so adding an underlay there makes no visible difference.